docs/guide/setup.md

```diff
@@ -41,6 +41,10 @@ declare module "*.vue" {
 }
 ```
 
+::: tip
+You can place this file in the root directory of your project or a directory named `@types`. You can place it in a custom directory, but you'll need to configure [`typeRoots`](https://www.typescriptlang.org/docs/handbook/tsconfig-json.html#types-typeroots-and-types) in the `tsconfig.json` file.
```

Comment:
> directory named `@types`

I know that this is what Kevin said on Discord, but I haven't seen any packages using `@types` directory. It's typically just `types`. `node_modules/@types/package-name` is a thing but that's not the same as having `@types` directory in your project.

I also usually put all this stuff in `./types` and typescript seems to pick those up without having to add them to `tsconfig.json` although documentation at https://www.typescriptlang.org/docs/handbook/tsconfig-json.html#types-typeroots-and-types doesn't seem to mention that being supported.

Reply:
So I should change `@types` to `types`? And if you want to name it `my-custom-types`, you still need that configuration, right?

Reply:
@rchl Yeah I think `@types` is kind of hidden auto detected folder by TypeScript, likely as `./types` I guess ?

Reply:
But then I would think it's common to use `types` directory while `@types` would look weird in a normal project.

Reply:
I changed it.